be/src/exec/olap_scan_node.cpp

```diff
@@ -1000,7 +1001,8 @@ std::pair<bool, void*> OlapScanNode::should_push_down_eq_predicate(doris::SlotDe
 
 template <typename T, typename ChangeFixedValueRangeFunc>
 Status OlapScanNode::change_fixed_value_range(ColumnValueRange<T>& temp_range, PrimitiveType type,
-                                              void* value, const ChangeFixedValueRangeFunc& func) {
+                                              void* value, const ChangeFixedValueRangeFunc& func,
+                                              PrimitiveType to_type) {
```

Comment:
This argument is added in order to meet patterns like `cast(datev2 as datetimev2)`. Since datetimev2 has not been  supported, this argument is indeed not used now.